torchgeo/models/resnet.py

```diff
@@ -52,6 +52,13 @@
     data_keys=["image"],
 )
 
+# https://github.com/microsoft/torchgeo/blob/8b53304d42c269f9001cb4e861a126dc4b462606/torchgeo/datamodules/ssl4eo_benchmark.py#L43 # noqa: E501
+_ssl4eo_l_transforms = AugmentationSequential(
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)),
+    K.RandomResizedCrop((224, 224), scale=(0.6, 1.0)),
```

Comment:
I think center crop would be better here. The idea behind these transforms is if you start with a pre-trained model, these are the transforms needed to ensure that the input images match the format expected by the model at **prediction** time, not at **training** time. At least, that's how I think of them, I could be wrong.

Reply:
Still needs to be updated